src/schema/nmdc.yaml

```diff
@@ -1081,6 +1081,7 @@ classes:
       - sample_link
 
       ## EMSL slots
+      - bulk_elect_conductivity
```

Comment:
will you be moving all existing EMSL slots underneath this comment?

Reply:
I think it's useful to organize, so we know where it came from. The new slots will go here.. most of the slots are MIxS & already exist